src/lib.rs

```diff
@@ -1,4 +1,5 @@
 use std::fs;
+use std::process::Command;
 use zed_extension_api::{self as zed, LanguageServerId, Result, settings::LspSettings};
```

Comment:
```suggestion
use zed_extension_api::{
    self as zed, LanguageServerId, Result, process::Command, settings::LspSettings,
};
```

`std::process::Command` is not supported in Zed extension, use `zed_extension_api::process::Command` instead.